package.json

```diff
@@ -23,6 +23,7 @@
     "gen": "lerna run --stream --scope @blocknote/dev-scripts gen",
     "start:built": "npx serve playground/dist -c ../serve.json",
     "build": "npm run clean && npm run gen && npm run clean && lerna run --stream build --concurrency 1",
+    "prepare": "npm run clean && npm run gen && npm run clean && lerna run --stream build --concurrency 1",
```

Comment:
I added the prepare script and installed BlockNote from my GitHub repository. Here's an example:

```json
  "dependencies": {
    "@blocknote/core": "git+https://github.com/kody-liou/BlockNote.git#Remove-some-non-null-assertion-operator",
    "@blocknote/mantine": "git+https://github.com/kody-liou/BlockNote.git#Remove-some-non-null-assertion-operator",
    "@blocknote/react": "git+https://github.com/kody-liou/BlockNote.git#Remove-some-non-null-assertion-operator",
   }
```
However, it throws an error:
 ```
 9:37:59 AM [vite] Internal server error: Failed to resolve import "@blocknote/core/fonts/inter.css" from "src/renderer/src/pages/home.tsx". Does the file exist?
  Plugin: vite:import-analysis
  File: D:/_src/personal/note-nook/src/renderer/src/pages/home.tsx:15:7
  30 |  import { useLocation } from "wouter";
  31 |  import { TagNavigator, Tags, useTagHook } from "@renderer/components/tag-navigator";
  32 |  import "@blocknote/core/fonts/inter.css";
     |          ^
  33 |  import "@blocknote/mantine/style.css";
  34 |  import { Editor, useEditorHook } from "../components/editor";
      at formatError (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:52009:46)
      at TransformContext.error (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:52003:19)
      at normalizeUrl (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:67306:33)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:67461:47
      at async Promise.all (index 8)
      at async TransformContext.transform (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:67382:13)
      at async Object.transform (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:52318:30)
      at async loadAndTransform (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:55073:29)
      at async viteTransformMiddleware (file:///D:/_src/personal/note-nook/node_modules/.pnpm/vite@5.2.13_@types+node@20.14.2/node_modules/vite/dist/node/chunks/dep-DEPSZ3SS.js:64920:32)
dep-DEPSZ3SS.js:13348
 ```